src/System.Net.Http/src/System/Net/Http/DiagnosticsHandler.cs

```diff
@@ -79,7 +72,11 @@ internal static bool IsEnabled()
             Activity currentActivity = Activity.Current;
             if (currentActivity != null)
             {
-                request.Headers.Add(DiagnosticsHandlerLoggingStrings.RequestIdHeaderName, currentActivity.Id);
+                // let ApplicaitonInsights set Request-Id
+
+                request.Headers.Add("traceparent", currentActivity.W3CId);
+                request.Headers.Add("tracestate", currentActivity.Tracestate);
```

Comment:
I am unclear on the value of ITextPropagationFormat.Inject if you are not using it here.   

By the way I am not bought into the value of ITextPropagationFormat  (seems like we will stop fiddling with headers soon enough at which point it serves no purpose, and besides we already have a more powerful hook (HttpMessageHandler) that we should probably leverage instead.